core/templates/dev/head/pages/exploration_editor/ExplorationEditor.js

```diff
@@ -388,6 +388,13 @@ oppia.controller('ExplorationEditor', [
         '</ul>')
     }];
 
+    // Remove save from tutorial if user does not has edit rights for
+    // exploration since in that case Save Draft button will not be visible
+    // on the create page.
+    if (!GLOBALS.can_edit) {
+      $scope.EDITOR_TUTORIAL_OPTIONS.splice(9, 1);
```

Comment:
@bansalnitish, Are we going to consider this as a complete tutorial?

(Note: we use to register this i.e, ( `core.controller.editor.StartedTutorialEventHandler` and `SiteAnalyticsService.registerFinishTutorialEvent`  and disables auto-tutorial popups.)

Reply:
I think we can consider it as a complete tutorial because since a user has opened an exploration for which they don't have edit rights, there is no save button visible and so I don't think, there is a point of providing a tutorial for a button which is not visible. Thanks!